src/ImagineFormat.jl

```diff
@@ -70,9 +72,18 @@ function load(io::Stream{format"Imagine"}; mode="r")
         axisnames = (axisnames..., :time)
         pixelspacing = (pixelspacing..., h["idle time between stacks"]+h["frames per stack"]*h["exposure time"])
     end
-#    ImageMeta(AxisArray(data, axisnames, pixelspacing), imagineheader=h, suppress=Set(Any["imagineheader"]))  # TODO: switch to this with julia 0.6.0
     axs = map((n,s,l)->Axis{n}(linspace(0*s, (l-1)*s, l)), axisnames, pixelspacing, size(data))
-    ImageMeta(AxisArray(data, axs), imagineheader=h, suppress=Set(Any["imagineheader"]))
+    is_bidi = h["bidirectional"]
```

Comment:
It's a required field from now on, but you're right that this will break loading old headers...good catch!  I'm less concerned about the old "bidirection" name, but I'll also support that just to be safe.